content/concepts/consensus-network/consensus-research.md

```diff
@@ -0,0 +1,9 @@
+# Consensus Research
+
+Ripple researches both the theoretical and the practical limits of the XRP Ledger's consensus protocols, and explores other ideas in the same space. The following table lists scholarly articles published by Ripple:
+
+| Date | Title | Authors | Summary |
+|---|---|---|---|
+| 2018-02-20 | [Cobalt: BFT Governance in Open Networks](https://arxiv.org/abs/1802.07240) | MacBrough | Introduces a novel atomic broadcast algorithm called Cobalt that allows more flexibility in consensus UNLs. |
```

Comment:
these external, PDF links aren't marked as such (yet) but that's awkward to do in Markdown so I'm skipping that for now